controllers/controller.js

```diff
@@ -0,0 +1,13 @@
+const selectAllTopics = require('../models/model.js');
+
+function getAllTopics(req, res, next){
+
+selectAllTopics().then((topics)=> {
+ 
+res.status(200).send(topics.rows)
```

Comment:
The deconstructing of rows should arguably happen in the model. Also to follow convention (and make your API easier to interpret) - ensure this is sent back in an object on an appropriately-named key